packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/metadata.py

```diff
@@ -40,6 +42,15 @@ class ParameterMetadata(NamedTuple):
     Required when 'type' is Kind.TypeMessage. Ignored for any other 'type'.
     """
 
+    def sanitized_display_name(self) -> str:
```

Comment:
Use `@property` and simplify the docstring (no "Returns:").

This codebase has type hints, so we don't have to duplicate the types in the docstring.